README.rst

```diff
@@ -26,7 +26,15 @@ A library to simplify use of Autograph
 Features
 --------
 
-* TODO
+SignatureVerifier
+=================
+
+The canonical implementation of certificate chain validation. Although
+some other implementations seem to exist (such as
+https://github.com/river2sea/X509Validation,
+https://github.com/alex/x509-validator, and
+https://github.com/openstack/cursive), all are marked as
+pre-production and/or needing work, so just do it ourselves.
```

Comment:
This makes me wonder if this part should be a separate package, or do you not want to commit to having something that's more maintained/production-ready than the above packages?

Reply:
I think the second. Specifically, I don't want to commit to something that's more general than the above packages. I'm comfortable building something that responds to the requirements of Mozilla when verifying Autograph signatures, but I'm not confident about building something that validates x5us in a general sense.